tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -82,7 +82,7 @@ def _prepare_state(balances_fn: Callable[[Any], Sequence[int]], threshold_fn: Ca
         # TODO: instead of upgrading a test phase0 genesis state we can also write a phase1 state helper.
         # Decide based on performance/consistency results later.
         state = phases[PHASE1].upgrade_to_phase1(state)
-    elif spec.fork == LIGHTCLIENT_PATCH and not fork_test:  # do not upgrade if spec ttttest
+    elif spec.fork == LIGHTCLIENT_PATCH and not is_fork_test:  # do not upgrade if spec ttttest
```

Comment:
We'll need to make this more generic eventually when doing fork tests for next fork, but... probably fine for now
```suggestion
    elif spec.fork == LIGHTCLIENT_PATCH and not is_fork_test:  # do not upgrade if spec test
```